.github/workflows/run_upd_dependency.yaml

```diff
@@ -0,0 +1,59 @@
+on:
+  workflow_call:
+
+jobs:
+  update_package_json:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install jq for JSON processing
+        run: sudo apt-get -y install jq
+
+      - name: Check updates Kaltura dependencies
+        run: |
+          # Check and update a dependency
+          checkAndUpdateDependency() {
+            local dependencyName="$1"
+            local packageName="$2"
+            local currentContent=$(cat package.json)
+            local currentVersion=$(echo "$currentContent" | jq -r ".kaltura.dependencies.\"$dependencyName\"")
+
+            # Check if the dependency exists in package.json
+            if [[ "$currentVersion" != "null" ]]; then
+              # Get the latest version of the package using npm
+              local latestVersion=$(npm view "$packageName" version)
+              
+              # Update the package.json file only if the version has changed
+              if [[ "$currentVersion" != "$latestVersion" ]]; then
+                echo "Update $dependencyName to version $latestVersion"
+                UPDATED_CONTENT=$(echo "$currentContent" | jq ".kaltura.dependencies.\"$dependencyName\" = \"$latestVersion\"")
+                echo "$UPDATED_CONTENT" > package.json
+                echo "COMMIT_REQUIRED=true" >> $GITHUB_ENV
+              else
+                echo "Dependency $dependencyName already has latest version: $latestVersion"
+              fi
+            else
+              echo "No $dependencyName dependency found in package.json"
+            fi
+          }
+
+          # Check and update playkit-kaltura-cuepoints dependency
```

Comment:
is there an option for the dependencies to be upgraded to not be hardcoded but to be supplied from outside ?

Reply:
addressed